commit.go

```diff
@@ -37,5 +67,20 @@ func (b *Builder) Commit(dest types.ImageReference, options CommitOptions) error
 		return err
 	}
 	err = copy.Image(policyContext, dest, src, getCopyOptions())
+	switch dest.Transport().Name() {
+	case storage.Transport.Name():
+		tags, err := expandTags(options.AdditionalTags)
+		if err != nil {
```

Comment:
Yeah, it should be doing that for the SetNames() call, at least.  I'll fix that.